### PR TITLE
Add clean up capability

### LINKED ISSUE #19: Add cleanup feature
- cleanup home (delete .helm folder)
- cleanup releases (delete tiller releases)
- uninstall Tiller

By default `--dry-run` is set, so no cleanup can be done without providing `--disable-dry-run` flag.
We do not want users by mistake to run the full clean up.

### PR DESCRIPTION
**Pull request by hickeyma**
It cleans up/removes Helm v2:
- Configuration: Helm v2 home directory (default)
- Release data: Tiller namespace, Tiller owner (default)
- Tiller deployment: Tiller namespace (default)

Closes #19

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,6 +64,41 @@ Flags:
...
+  -t, --tiller-ns string         namespace of Tiller (default "kube-system")
+      --tiller-out-cluster       when  Tiller is not running in the cluster e.g. Tillerless
+```
+
+It will clean:
+- Configuration (Helm home directory)
+- v2 release data
+- Tiller deployment
+
+For cleanup it uses default Helm v2 home and v3 config and data folders.
```
Comment (rimusz) [inline README.md]: It should be only: For cleanup it uses default Helm v2 home folder.

File: README.md [hunk 2]
```diff
@@ -64,6 +64,41 @@ Flags:
...
+To override those folders you need to set environment variables `HELM_V2_HOME`, `HELM_V3_CONFIG` and `HELM_V3_DATA`:
```
Comment (rimusz) [inline README.md]: change to: To override that folder you need to set environment variable `HELM_V2_HOME`:

File: README.md [hunk 3]
```diff
@@ -64,6 +64,41 @@ Flags:
...
+
+```console
+$ export HELM_V2_HOME=$PWD/.helm2
+$ export HELM_V3_CONFIG=$PWD/.helm3
```
Comment (rimusz) [inline README.md]: remove it

File: README.md [hunk 4]
```diff
@@ -64,6 +64,41 @@ Flags:
...
+$ export HELM_V3_DATA=$PWD/.helm3
```
Comment (rimusz) [inline README.md]: remove it

File: README.md [hunk 5]
```diff
@@ -64,6 +64,41 @@ Flags:
...
+$ helm 2to3 move config
```
Comment (rimusz) [inline README.md]: change to: helm 2to3 cleanup

**Comment by rimusz** [comment]
readme needs some changes.
I build and tested code with all three commands:
```
  cleanup     cleanup Helm v2 configuration, release data and Tiller deployment
  convert     migrate Helm v2 release in-place to Helm v3
  move.       migrate Helm v2 configuration in-place to Helm v3
```
everything worked as expected 👍

**Comment by rimusz** [comment]
please also bump the plugin version.

File: README.md (hunk 1 above)
Comment (hickeyma) [reply README.md]: done

File: README.md (hunk 3 above)
Comment (hickeyma) [reply README.md]: done

File: README.md (hunk 5 above)
Comment (hickeyma) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (hickeyma) [reply README.md]: done

File: README.md (hunk 4 above)
Comment (hickeyma) [reply README.md]: done

**Comment by hickeyma** [comment]
@rimusz Thank you for the review. I have updated the comments. It is ready for review again.

**Comment by rimusz** [comment]
LGTM

**Pull request merged**